newIDE/app/src/MainFrame/index.js

```diff
@@ -327,13 +348,29 @@ class MainFrame extends React.Component<Props, State> {
         );
       },
       err => {
-        showErrorBox(
-          i18n._(
-            t`Unable to open this project. Check that the path/URL is correct, that you selected a file that is a game file created with GDevelop and that is was not removed.`
-          ),
-          err
-        );
-        return;
+        if (
+          shouldOpenAutosave &&
+          shouldOpenAutosave(projectFilePath, autoSavePath, false)
+        ) {
+          //eslint-disable-next-line
+          const answer = confirm(
+            i18n._(
+              `The project file appears to be malformed, but an *.autosave exists.\nWould you like to try to load it instead?`
+            )
+          );
+          if (answer) {
+            this.openFromPathOrURL(autoSavePath);
+            this.openProjectManager(true);
```

Comment:
Why not using this.openSceneOrProjectManager() like other place where openFromPathOrURL is called?

Reply:
ah yes you're right. Thank you for changing it